runtime/lua/vim/filetype.lua

```diff
@@ -770,6 +770,14 @@ local extension = {
   mm = function() vim.fn["dist#ft#FTmm"]() end,
   mms = function() vim.fn["dist#ft#FTmms"]() end,
   p = function() vim.fn["dist#ft#FTprogress_pascal"]() end,
+  patch = function(path, bufnr)
+    local firstline = vim.fn.getline(1)
+    if string.find(firstline, "From ") and string.find(firstline, "Mon Sep 17 00:00:00 2001") then
```

Comment:
I don't understand why this pattern is used, but it's what filetype.vim does so ¯\\\_(ツ)\_/¯ 

```suggestion
    if string.find(firstline, "^From " .. string.rep("%x", 40) .. "+ Mon Sep 17 00:00:00 2001$") then
```